MDANSE/Src/MDANSE/Framework/QVectors/IQVectors.py

```diff
@@ -29,16 +30,28 @@
     from MDANSE.Framework.OutputVariables.IOutputVariable import OutputData
     from MDANSE.MolecularDynamics.UnitCell import UnitCell
 
+GAUSS_WIDTH_FACTOR = 2 * 2.35482  # 2*sqrt(2*ln(2))
```

Comment:
May as well go whole hog on the constant:

```suggestion
GAUSS_WIDTH_FACTOR = 4.70964  # 2*sqrt(2*ln(2))
```